src/EditorFeatures/CSharpTest/Formatting/Indentation/SmartIndenterTests.cs

```diff
@@ -418,7 +418,7 @@ void Method()
             AssertSmartIndent(
                 code,
                 indentationLine: 7,
-                expectedIndentation: null,
+                expectedIndentation: 4,
```

Comment:
we no longer ever fail to produce indentation from the IIndentationSerevice.

Reply:
❕ The comment needs to be updated to match

Reply:
true!